### PR TITLE
A few fixes

### PR DESCRIPTION
**Pull request by claudiodsf**
- ~~Use `traceid` in file names, so that it is possible to rerun the notebook for a different stream~~
- Fix typos
- Added `.gitignore`
- Added back `channelcode` into the plot title
- Added ability to edit PDF text in Illustrator
- Added `conda-forge` in installation instructions.

Also, to reduce the diffs, I think it's better to commit the notebooks after having cleared them.
We can think at adding a rendered version, saved as HTML to show the results. What do you think?

### CODE REVIEW

**Comment by ThomasLecocq** [comment]
agree on all :) the diffs are getting crazy here :)

**Comment by FMassin** [comment]
This is overwriting a lot of my addons, could you merge with last version?

**Comment by claudiodsf** [comment]
Oops, sorry, I didn't noticed that you already fixed some of these things.

I'll solve the conflicts and rebase 😉

**Comment by FMassin** [comment]
> Oops, sorry, I didn't noticed that you already fixed some of these things.
> 
> I'll solve the conflicts and rebase 😉

OK, see above and #5.

**Comment by claudiodsf** [comment]
Ok, done!

File: SeismoSocialDistancing.ipynb [hunk 1]
```diff
@@ -325,7 +325,7 @@
     "    plt.gca().yaxis.set_major_formatter(ticks)\n",
     "    plt.ylabel(\"Displacement (nm)\")\n",
     "\n",
-    "    plt.title('Seismic Noise for %s - Filter: [%s] Hz' % (main,\n",
```
Comment (FMassin) [inline SeismoSocialDistancing.ipynb:593]: Hey @claudiodsf , you spotted something I got wrong, I meant to do:
```
"    plt.title('Seismic Noise for %s - Filter: [%s] Hz' % (channelcode[-3:]+main,\n",
```
To it adapts to 3-components plots as well as single component...

Comment (claudiodsf) [reply SeismoSocialDistancing.ipynb:593]: Ok, I'll fix it and force push

Comment (FMassin) [reply SeismoSocialDistancing.ipynb:593]: Great ! Thanks

**Comment by FMassin** [comment]
> * ~Use `traceid` in file names, so that it is possible to rerun the notebook for a different stream~
I did not meant to get rid of this, I like it, and it is not conflicting at all :), I would keep it.

> Also, to reduce the diffs, I think it's better to commit the notebooks after having cleared them.
> We can think at adding a rendered version, saved as HTML to show the results. What do you think?
Yes, I think you can do this within a single cell added at the end of the notebook:
```
%autosave 1
import time
time.sleep(2)
%autosave 120
!jupyter nbconvert --to html SeismoSocialDistancing.ipynb
!nbstripout SeismoSocialDistancing.ipynb
```
It looks awful but it works.

**Comment by claudiodsf** [comment]
> I did not meant to get rid of this, I like it, and it is not conflicting at all :), I would keep it.

Ok, see my last commit.

**Comment by claudiodsf** [comment]
> Yes, I think you can do this within a single cell added at the end of the notebook:

I'll look into that tomorrow 🥱

**Comment by megies** [comment]
> agree on all :) the diffs are getting crazy here :)

One of the downsides of Jupyter notebooks, not good for version control. Has been one of the main issues for seismo-live all the time, that's why we recently switched to keeping notebooks as jupytext in the version control and build notebooks just for the final page from these. See krischer/seismo_live#33

**Comment by claudiodsf** [comment]
@ThomasLecocq  Can we merge this? I'll look into remaining issues in a different PR.

**Comment by ThomasLecocq** [comment]
sure ! don't make the whole calculation stuff tooooooo crazy, msnoise2.0 (actually in master later today) has the QC/PPSD computation built in, all that will be required will be

* msnoise db init
* python get_your_data.py
* msnoise scan_archive --path /path/ --recursively
* msnoise new_jobs
* msnoise qc compute_psd

**Pull request merged**